src/runtime_src/core/pcie/driver/linux/xocl/subdev/icap.c

```diff
@@ -1228,6 +1230,9 @@ static void icap_clean_axlf_section(struct icap *icap,
 	case IP_LAYOUT:
 		target = (void **)&icap->ip_layout;
 		break;
+	case SOFT_KERNEL:
```

Comment:
Could we define a macro like:
#define PS_KERNEL SOFT_KERNEL

Then we can use PS_KERNEL in the code :) Since you will use 'ps kernel' every where.

Reply:
Thanks. That is a good suggestion.

When we decide to change the name from softkernel to PS kernel, we agree to gradually clean up the name inside our code. Let's follow up with a discussion how we want to proceed from xma, xrm, xclbin, xrt libs and kernel drivers